pages/main.js

```diff
@@ -55,11 +55,20 @@ function createTable (dats, send) {
         ? Math.round(((stats.blocksProgress / stats.blocksTotal) | 0) * 100)
         : 100
       : 0
+    const state = progress < 1
+      ? dat.stats.peers > 0
+        ? 'loading'
+        : 'paused'
+      : 'complete'
     return html`
       <tr class="dat-list__item">
         <td>
           <div class="dat-hexagon">
-            ${dat.owner ? '↑' : '↓'}
+            ${{
+              loading: '↓',
+              paused: '',
+              complete: '↑'
+            }[state]}
```

Comment:
it's something I learned from @substack years ago, once you get used to it it's a lot easier to read and type than say

```js
${
  state === 'loading' && 'v' ||
  state === 'paused' && '' ||
  state === 'complete' && '^'
}
```

it's basically a switch statement as a value, which JavaScript is missing.